src/storage/config.rs

```diff
@@ -276,21 +277,11 @@ impl BlockCacheConfig {
         }
     }
 
-    pub fn build_shared_cache(&self, engine_type: EngineType) -> Cache {
+    pub fn build_shared_cache(&self) -> Cache {
         if self.shared == Some(false) {
             warn!("storage.block-cache.shared is deprecated, cache is always shared.");
         }
-        let capacity = match self.capacity {
-            None => {
-                let total_mem = SysQuota::memory_limit_in_bytes();
-                if engine_type == EngineType::RaftKv2 {
-                    ((total_mem as f64) * RAFTSTORE_V2_BLOCK_CACHE_RATE) as usize
-                } else {
-                    ((total_mem as f64) * BLOCK_CACHE_RATE) as usize
-                }
-            }
-            Some(c) => c.0 as usize,
-        };
+        let capacity = self.capacity.unwrap_or(FALLBACK_BLOCK_CACHE_CAPACITY).0 as usize;
```

Comment:
It means we have to set the cache size manually?

Reply:
Yes

Reply:
See the comment for `FALLBACK_BLOCK_CACHE_CAPACITY`. It's set in validate.